tests/univariate/test_KDEUnivariate.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `univariate` package."""
+
+from unittest import TestCase
+
+import numpy as np
+
+from copulas.univariate.KDEUnivariate import KDEUnivariate
+
+
+class TestKDEUnivariate(TestCase):
+    def setup_norm(self):
+        """set up the model to fit standard norm data"""
+        self.kde = KDEUnivariate()
+        # use 42 as a fixed random seed
+        np.random.seed(42)
+        column = np.random.normal(0, 1, 1000)
+        self.kde.fit(column)
+
+    def test___init__(self):
+        """On init, model are set to None."""
+        self.kde = KDEUnivariate()
+        assert self.kde.model is None
+
+    def test_fit_uniform(self):
+        """On fit, kde model is instantiated with passed data"""
+        self.kde = KDEUnivariate()
+        self.kde.fit([1, 2, 3, 4, 5])
+        assert self.kde.model is not None
```

Comment:
Here you are not testing that the model is instantiated with data, you are just looking that the attribute `model` of the instance is not `None`.

You should check that model is indeed a  `scipy.stats.gaussian_kde` instance. And check also the attribute `dataset`of the model is the same array you passed to `fit`